helm/hpcc/templates/dali.yaml

```diff
@@ -34,15 +34,20 @@ spec:
         volumeMounts:
 {{ include "hpcc.addConfigMapVolumeMount" $dali | indent 8 }}
 {{ include "hpcc.addDaliVolumeMount" $commonCtx | indent 8 }}
-{{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" ) ) | indent 8 }}
+{{- $secretsCategories := list "system" }}
 {{- range $sashaName, $_sasha := $dali.services -}}
 {{- $sasha := ($_sasha | default dict) -}}
+{{- $secretsCategories2 := list "system" }}
```

Comment:
for clarity should rename this sashaServiceSecretCategories vs the one above, which I'd rename daliSecretsCategories